### PR TITLE
CODE-2790: clean up PCGIOHandler and PCGFile

### PR DESCRIPTION
**Pull request by grimreaper**
- remove unused functions
    - rename some variables
    - add more final

### CODE REVIEW

**Comment by thpr** [comment]
This apparently overlaps PR#640 since it also include IIFNode

File: code/src/java/pcgen/io/PCGIOHandler.java [hunk 1]
```diff
@@ -257,7 +257,7 @@ public void read(PlayerCharacter pcToBeRead, InputStream in,
 		}
 	}
 
-	private boolean isPCGCersion2(List<String> lines)
+	private static boolean isPCGCersion2(List<String> lines)
```
Comment (thpr) [inline code/src/java/pcgen/io/PCGIOHandler.java]: @jdempsey-au What do you think here?  I don't think we have a standard on this.  Honestly, I'm ambivalent to having things being static or not (don't have a strong opinion), so not sure it matters, but if things are going to be cleaned up, we should decide if this is worth the effort to be changed.

Comment (jdempsey-au) [inline code/src/java/pcgen/io/PCGIOHandler.java]: I'd leave them as instance methods unless there is a notable benefit to having them static.

**Comment by thpr** [comment]
@grimreaper Please remove the two static items, and please resolve the conflicts created when the previous IIF PR was merged.  Once those are complete, we can merge this PR.

File: code/src/java/pcgen/io/PCGIOHandler.java (hunk 1 above)
Comment (grimreaper) [inline code/src/java/pcgen/io/PCGIOHandler.java]: Why should they be instance methods?  It acts a lot more like a helper method than a property of the object itself.  Perhaps I'm wrong?

**Comment by LegacyKing** [comment]
@grimreaper did you get the last note? Conflicts and corrections so we can merge this. Thanks!

**Comment by LegacyKing** [comment]
This conflicted and no reply for almost a month.

**Comment by LegacyKing** [comment]
@grimreaper Eitan, any update with this???

**Comment by LegacyKing** [comment]
Closing this out as Eitan has already closed the other PR. I've resubmitted a fixed/non conflicted version.

**Pull request closed without merging**